data/json/effects.json

```diff
@@ -1652,5 +1652,12 @@
         "remove_message": "The broken limb has mended.",
         "max_duration": 2,
         "rating": "bad"
-    }
+    },
+    {
+        "type": "effect_type",
+        "id": "laidegg",
+        "name": [ "Laid an egg" ],
+        "desc": [ "This creature has laid an egg recently" ],
```

Comment:
There should be a period. Some translators complained to me about the inconsistency of the period symbols at the end of the strings in game recently so I'm paying attention to that in new PRs.